docs/content/en/docs/reference/baremetal/bare-preparation.md

```diff
@@ -5,3 +5,83 @@ weight: 20
+* Make sure the following settings are in place:
+  * UEFI is enabled on all target cluster machines
+  * PXE boot is enabled on each NIC
+  * PXE is set as the first device in each machine's boot order
+  * IPMI over LAN is enabled on the IPMI interfaces
+* Go to the IPMI settings for each machine and set the IP address (bnc_ip), username (bnc_username), and password (bnc_password) to use later in the CSV file.
+
+## Prepare hardware inventory
+Create a CSV file to provide information about all physical machines that you are ready to add to your target Bare Metal cluster.
+This file will be used:
+
+* When you generate the hardware file to be included in the cluster creation process described in the [Create Bare Metal production cluster]({{< relref "../../getting-started/production-environment" >}}) Getting Started guide.
+* To provide information that is passed to each machine from the Tinkerbell DHCP server when the machine is initially PXE booted.
+
+The following is an example of an EKS Anywhere Bare Metal hardware CSV file:
+
+```
+hostname,bmc_ip,bmc_username,bmc_password,mac,ip_address,netmask,gateway,nameservers,labels,disk
+eksa-cp01,10.10.44.1,root,PrZ8W93i,CC:48:3A:00:00:01,10.10.50.2,255.255.254.0,10.10.50.1,X.X.X.X,,/dev/sda
+eksa-cp02,10.10.44.2,root,Me9xQf93,CC:48:3A:00:00:02,10.10.50.3,255.255.254.0,10.10.50.1,X.X.X.X,,/dev/sda
+eksa-wk01,10.10.44.3,root,Z8x2M6hl,CC:48:3A:00:00:03,10.10.50.4,255.255.254.0,10.10.50.1,X.X.X.X,,/dev/sda
+eksa-wk02,10.10.44.4,root,B398xRTp,CC:48:3A:00:00:04,10.10.50.5,255.255.254.0,10.10.50.1,X.X.X.X,,/dev/sda
+eksa-wk03,10.10.44.5,root,w7EenR94,CC:48:3A:00:00:05,10.10.50.6,255.255.254.0,10.10.50.1,X.X.X.X,,/dev/sda
+
+```
+
+The CSV file is a comma-separated list of values in a plain text file, representing physical control plane and worker machines (not virtual machines) in your cluster.
+Multiple values in an item can be separated by pipe symbols (|).
+
+The following sections describe each value.
+
+### hostname
+The hostname assigned to the machine.
+### bnc_ip
+The IP address assigned to the IPMI interface on the machine.
+### bmc_username
+The username assigned to the IPMI interface on the machine.
+### bnc_password
+The password associated with the `bmc_username` assigned to the IPMI interface on the machine.
+### mac
+The MAC address of the network interface card (NIC) that provides access to the host computer.
+### ip_address
+The IP address providing access to the host computer.
+### netmask
+The netmask associated with the `ip_address` value.
+In the example above, a /23 subnet mask is used, allowing you to use up to 510 IP addresses in that range. 
+### gateway
+IP address of the interface that provides access (the gateway) to the Internet.
+### nameservers
+The IP address of the server that you want to provide DNS service to the cluster.
+### labels
+Optional labels you can set on each node that are added to any labels set on that node by the cluster.
```

Comment:
Labels are not added to the 'node' per say. This would be a bit confusing. These labels are only added to the 'Hardware' CRD in the cluster, not the Kubernetes node.

Reply:
Okay. I changed "node" to "machine" to make that clearer.